### PR TITLE
Prolog now completion only provides valid attributes

### LINKED ISSUE #573: Prolog completion of encoding tag leads to "No more pseudo attributes are allowed"
```xml
<?xml version="1.0" standalone="no" encoding="UTF-8"  ?>
<foo>bar</foo>
```

yields:

Screen Shot 2019-10-15 at 6 29 01 PM

### PR DESCRIPTION
**Pull request by NikolasKomonen**
Minor fix to NoMorePseudoAttributes error range.

Fixes #573

Signed-off-by: Nikolas Komonen <nikolaskomonen@gmail.com>

### CODE REVIEW

File: org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/prolog/PrologModel.java [hunk 1]
```diff
@@ -201,4 +220,71 @@ private static void createCompletionItemsForValues(Collection<String> enumeratio
...
+	 * <element a="1" b="2" | c="3">
+	 * 
+	 * This will return 2 since if you insert a new attribute there you can access
+	 * it from the list of attributes with this index.
+	 * @param completionOffset
+	 * @param element
+	 * @return
+	 */
+	private static int getAttributeCompletionPosition(int completionOffset, DOMNode element) {
+		int attributeListLength;
```
Comment (angelozerr) [inline org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/prolog/PrologModel.java]: remove this declaration

File: org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/prolog/PrologModel.java [hunk 2]
```diff
@@ -201,4 +220,71 @@ private static void createCompletionItemsForValues(Collection<String> enumeratio
...
+		
+		NamedNodeMap attributeList = element.getAttributes();
+		
+		if(attributeList == null) {
+			return 0;
+		}
+		else {
```
Comment (angelozerr) [inline org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/prolog/PrologModel.java]: remove the else and write 
int attributeListLength = attributeList.getLength()

**Comment by NikolasKomonen** [comment]
@angelozerr updated

**Comment by angelozerr** [comment]
Works like a charm @NikolasKomonen

**Pull request merged**